.github/workflows/build_all.yml

```diff
@@ -667,7 +545,6 @@ jobs:
           BUILD_METADATA: '{"github:workflow": "${{ github.workflow }}", "github:run_id": "${{ github.run_id }}", "github:actor": "${{ github.actor }}", "github:repository": "${{ github.repository }}",
             "github:event_name": "${{ github.event_name }}", "github:sha": "${{ github.sha }}", "github:sha:url": "${{github.event.repository.owner.html_url}}/magma/commit/${{github.sha}}",
             "github:ref": "${{ github.ref }}"}'
-          AGW_ARTIFACTS: ${{ needs.agw-build.outputs.artifacts }}
```

Comment:
for #15161 this functionality needs to be added to the bazel based debian build job